app/src/main/java/com/example/voyagerx/LandingPageFragment.kt

```diff
@@ -16,4 +21,14 @@ class LandingPageFragment : Fragment() {
         return inflater.inflate(R.layout.fragment_landing_page, container, false)
     }
 
-}
\ No newline at end of file
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        lifecycleScope.launchWhenResumed {
+            val response = LaunchRepository.getLaunches()
+
+            Log.d("Repository", "Success $response")
+        }
+    }
+
+}
```

Comment:
@chrsbell I added this to test the repository. I thought I would just leave it here in case you wanted to use it.

Reply:
Makes sense, thanks! This looks good to me as well.